build_debian.sh

```diff
@@ -260,6 +260,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
 ## Prepare empty directory to trigger mount move in initramfs-tools/mount_loop_root, implemented by patching
 sudo mkdir $FILESYSTEM_ROOT/host
 
+## Create loop device for /var/log to limit its size to 1000000*512=500MB
+sudo mkdir -p $FILESYSTEM_ROOT/var/disk-img
+sudo LANG=C chroot $FILESYSTEM_ROOT dd if=/dev/zero of=/var/disk-img/var-log.ext4 count=1000000
```

Comment:
we probably need to create per platform file under topdir/hwsku/$onie_platform/platform.conf

In that conf, we can specify the log disk size, and then at the installation time, based on the onie string, we create the log disk.

we may put some other things into the directory such as sensors.conf, pwmconfig.